src/python/grpcio/grpc/aio/_call.py

```diff
@@ -444,6 +444,8 @@ async def _write(self, request: RequestType) -> None:
                                                self._request_serializer)
         try:
             await self._cython_call.send_serialized_message(serialized_request)
+        except cygrpc.InternalError:
+            await self._raise_for_status()
```

Comment:
in my test this raises the AioRpcError (StatusCode.UNKNOWN) which then still gets caught in _consume_request_iterator and logged at debug

Reply:
This change and the original change is similar functional-wise. This is fine. Changing the `_write` function ensures we are catching all InternalError from the send/write message behavior.